core/libdeno/binding.cc

```diff
@@ -223,23 +223,29 @@ void Send(const v8::FunctionCallbackInfo<v8::Value>& args) {
 
   v8::HandleScope handle_scope(isolate);
 
+  int cmd_id = 0;
+  if (args[0]->IsNumber()) {
+    auto cmd_id_v8_handle = v8::Local<v8::Number>::Cast(args[0]);
+    cmd_id = cmd_id_v8_handle->Value();
```

Comment:
This may have a perf implication. Did you see any degradation in the bench?

Reply:
A little <= 5%

Reply:
Would encoding this into control be more efficient?

Reply:
Yea that's what currently happens.

One of the main reasons for this "op" design was the observation we had from Node, that once you're out of JS, basically any interaction with V8 handles is extremely slow. So we try to limit this as much as possible by encoding everything into the buffers.

Proper review soon - I've only given it a glance so far. 5% isn't much...

Reply:
Ok I might encode this to control in Deno.core.dispatch, and decode in pre_dispatch.

Reply:
I gave it a try and that seems to make things even slower not sure why though.